source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1266,6 +1267,28 @@ message TransformationPermutePhiOperands {
 
 }
 
+message TransformationPropagateInstructionDown {
+
+  // Propagates an instruction from |block_id| into its successors.
+  // Concretely, the transformation clones the propagated instruction
+  // into some of the predecessors of |block_id| and removes the original
+  // instruction. Additionally, OpPhi instruction may be added to make sure
+  // that the transformation can be applied in various scenarios.
+
+  // Id of the block to propagate an instruction from.
+  uint32 block_id = 1;
```

Comment:
Can you comment on how the instruction to be propagated will be decided upon given this block?  I suppose it is the final instruction in the block for which propagation is possible?

Reply:
Fixed in 69b41e9.